zxlive/dialogs.py

```diff
@@ -76,9 +76,9 @@ class ImportRuleOutput:
     file_path: str
     r: CustomRule
 
-def show_error_msg(title: str, description: Optional[str] = None) -> None:
+def show_error_msg(title: str, description: Optional[str] = None, parent: Optional[QWidget] = None) -> None:
```

Comment:
The changes in this file regarding the parent are related to a separate issue. Do you mind splitting this into a separate PR?